lib/formtools/plugins/plugins-helpers.js

```diff
@@ -533,9 +536,25 @@ var transformSortBy = function (labels, sortBy) {
 
     return sortBy.map(function (fieldName) {
 
+        // Add the ability to pass an object.
+        if (typeof fieldName !== 'string') {
+
+            if (!fieldName.field) {
+                throw new Error('You must provide a field');
+            }
+
+            return Object.assign({}, {
+                defaultOrder: 'asc',
+                label: (labels[fieldName.field] || fieldName.label),
```

Comment:
What if they haven't provided a label? It doesn't seem to default the label to the `fieldName` itself, if the field doesn't exist in the `label` array and the `label` wasn't provided in the `sortBy` object?

Reply:
@smebberson So should we just add another fallback `|| fieldName.field`?

Reply:
@allanchau, in fact, it already defaults when the label is missing. I think that is because the label array is defaulted with the `fieldname` if a label isn't provided. So I don't think it's necessary - you can leave it as is.